src/openrct2/world/Sprite.cpp

```diff
@@ -1,4 +1,4 @@
-/*****************************************************************************
+﻿/*****************************************************************************
```

Comment:
Again.

Can you please stop your IDE from inserting these silly BOMs, which should not be present in UTF-8 files?

Reply:
I have no bloody idea how it keeps doing this, I always put it back but at some point it just screws me over again.